wings/market/ddex_market.pyx

```diff
@@ -440,8 +456,61 @@ cdef class DDEXMarket(MarketBase):
             self.logger().debug(f"{data}")
             return data
 
+    async def _update_trade_fees(self):
+        cdef:
+            double current_timestamp = self._current_timestamp
+
+        if current_timestamp - self._last_update_trade_fees_timestamp > 60.0 * 60.0 or self._gas_fee_usd == NaN:
+            calc_fee_url = f"{self.DDEX_REST_ENDPOINT}/fees"
+            params = {
+                "amount": "1",
+                "marketId": "HOT-WETH",
+                "price": "1",
+            }
+            res = await self._api_request(http_method="get", url=calc_fee_url, params=params)
+            # maker / taker trade fees are same regardless of pair
+            self._maker_trade_fee = float(res["data"]["asMakerFeeRate"])
+            self._taker_trade_fee = float(res["data"]["asTakerFeeRate"])
+            # gas fee from api is in quote token amount
+            self._gas_fee_weth = float(res["data"]["gasFeeAmount"])
+            params = {
+                "amount": "1",
+                "marketId": "WETH-DAI",
+                "price": "1",
+            }
+            res = await self._api_request(http_method="get", url=calc_fee_url, params=params)
+            # DDEX charges same gas fee for both DAI and TUSD
+            self._gas_fee_usd = float(res["data"]["gasFeeAmount"])
+            self._last_update_trade_fees_timestamp = current_timestamp
+
+    def calculate_fees(self,
+                       trading_pair: str,
+                       amount: float,
+                       price: float,
+                       order_type: OrderType,
+                       order_side: TradeType) -> List[TradeFee]:
+        return self.c_calculate_fees(trading_pair, amount, price, order_type, order_side)
+
+    cdef list c_calculate_fees(self,
+                               str trading_pair,
```

Comment:
We use "symbol" to indicate trading_pair everywhere else. Should stay consistent here